_pages/appendix-skyve-application-configuration.md

```diff
@@ -59,6 +64,32 @@ Alternatively, you can place the API key in the project JSON file under `api` ->
 	api: {
 		googleRecaptchaSiteKey: "xxxxxxxxxxx",
 ```
+#### Cloudflare Turnstile
+
+First, sign up for a Cloudflare Turnstile key as follows:
```

Comment:
Can you just add a comment like (requires skyve > 9.1.0) then we can merge this in?